client/my-sites/upgrades/domain-management/style.scss

```diff
@@ -848,20 +848,18 @@ ul.email-forwarding__list {
 	}
 }
 
-.name-servers {
-	.icann-verification {
-		.button {
-			margin-bottom: 15px;
+.icann-verification {
```

Comment:
I think verification in the global css namespace is likely to have name clash in the future. We should namespace it.